src/main/java/org/tron/core/net/node/NodeDelegateImpl.java

```diff
@@ -149,18 +149,18 @@ public boolean handleTransaction(TransactionCapsule trx) throws BadTransactionEx
       logger.info("AccountResourceInsufficientException" + e.getMessage());
       return false;
     } catch (DupTransactionException e) {
-      logger.info("dup trans" + e.getMessage());
+      logger.info("Dup trans" + e.getMessage());
       return false;
     } catch (TaposException e) {
-      logger.info("tapos error" + e.getMessage());
+      logger.info("Tapos error" + e.getMessage());
       return false;
     } catch (ReceiptException e) {
       logger.info("Receipt exception," + e.getMessage());
     } catch (TooBigTransactionException e) {
-      logger.info("too big transaction" + e.getMessage());
+      logger.info("Too big transaction" + e.getMessage());
       return false;
     } catch (TransactionExpirationException e) {
-      logger.info("expiration transaction" + e.getMessage());
+      logger.info("Expiration transaction" + e.getMessage());
       return false;
     } catch (TransactionTraceException e) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Either log or rethrow this exception. ([squid:S1166](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1166))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1500&issues=AWXBxdl7ashydt7ke7Fb&open=AWXBxdl7ashydt7ke7Fb)

[ISSUE_KEY]: AWXBxdl7ashydt7ke7Fb